### PR TITLE
Sutton sc 1513 information page new template

### PR DESCRIPTION
**Pull request by seanjessen**
### Summary
https://app.shortcut.com/helpyourselfsutton/story/1513/information-page-new-template

### Development checklist
N/A

### Release checklist
N/A

### Notes
N/A

### CODE REVIEW

File: src/types/types.ts [hunk 1]
```diff
@@ -1,5 +1,34 @@
 import { IconName } from '@fortawesome/fontawesome-svg-core';
 
+export interface IPage {
+  id: string;
+  page_type: string;
+  title: string;
+  image: string;
+  introduction: {
+    copy: string[]
+  },
```
Comment (appsol) [inline src/types/types.ts:10]: The introduction is a node within the content field, not a field on it's own.

File: src/types/types.ts [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+      copy: string[],
+      title: string
+    },
+    introduction: null | {
+      copy: string[]
+    },
+  },
+  parent: [],
+  children: [],
+  collections: []
```
Comment (appsol) [inline src/types/types.ts:29]: collections has been replaced with collection_categories and collection_personas

File: src/views/Page/InformationPage.tsx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+          )}
+          {props.content.image && (
+            <div className="flex-col flex-col--12">
+              <img alt={props.content.title} className="information-page__image" src={props.content.image} />
+            </div>
+          )}
+          {props.content.content.introduction.copy && (
+            <div className="flex-col flex-col--12">
+              <ReactMarkdown
+                children={props.content.content.introduction.copy[0]}
```
Comment (appsol) [inline src/views/Page/InformationPage.tsx:46]: Should this not be props.content.introduction.copy[0] ?

**Review by appsol — CHANGES_REQUESTED** [review]
Just a couple of issues. I could just be reading your code incorrectly, but they may be errors.

File: src/views/Page/InformationPage.tsx (hunk 3 above)
Comment (seanjessen) [reply src/views/Page/InformationPage.tsx:46]: I think this is correct it's getting pushed via props and is outputting correctly

File: src/types/types.ts (hunk 2 above)
Comment (seanjessen) [reply src/types/types.ts:29]: Amended

**Review by appsol — APPROVED** [review]

**Pull request merged**